ppr-ui/src/views/userAccess/QsReviewConfirm.vue

```diff
@@ -9,36 +27,37 @@
       />
     </section>
 
-    <section id="qs-confirm-requirements" class="mt-15">
-      <h2>Confirm</h2>
-      <p class="mt-1">
-        The following requirements must be confirmed.
-      </p>
-      <ConfirmRequirements :validateReview="validateReview" />
-    </section>
-
-    <section id="qs-authorization" class="mt-15">
-      <h2>Authorization</h2>
-      <p class="mt-1">
-        Enter the legal name of the person authorized to complete and submit this application.
-        <b>Note:</b> The authorized person must be an active B.C. lawyer or notary in good standing.
-      </p>
-      <Authorization :validateReview="validateReview" />
-    </section>
+   <div class="increment-sections">
+     <section id="qs-confirm-requirements" class="mt-15">
+       <h2>Confirm</h2>
+       <p class="mt-1">
+         The following requirements must be confirmed.
+       </p>
+       <ConfirmRequirements :validateReview="validateReview" />
+     </section>
 
+     <section id="qs-authorization" class="mt-15">
+       <h2>Authorization</h2>
+       <p class="mt-1">
+         Enter the legal name of the person authorized to complete and submit this application.
+         <b>Note:</b> The authorized person must be an active B.C. lawyer or notary in good standing.
+       </p>
+       <Authorization :validateReview="validateReview" />
+     </section>
+   </div>
```

Comment:
The incremental wrapper, as mentioned above. Gives some control over what/where we increment.